pulsar-broker/src/test/java/org/apache/pulsar/broker/service/SystemTopicBasedTopicPoliciesServiceTest.java

```diff
@@ -201,6 +201,7 @@ public void testCacheCleanup() throws Exception {
         assertNotNull(listMap.get(topicName).get(0));
 
         admin.topics().deletePartitionedTopic(topic, true);
+        admin.namespaces().unload(NAMESPACE1);
```

Comment:
Why this change?

Reply:
This is the test for checking the cache cleanup, you can check the conversation with @315157973, the topic policy cache is based on the namespace, so we need to unload the namespaces to make sure the cache been reloaded, in the conversation with @315157973 I have mentioned I need to create a separate PR to handle the topic policy deletion when deleting the topic.